synnax/pkg/user/service.go

```diff
@@ -41,23 +55,18 @@ func (c Config) Validate() error {
 	return v.Error()
 }
 
-// Override implements config.Properties.
-func (c Config) Override(other Config) Config {
-	c.DB = override.Nil(c.DB, other.DB)
-	c.Ontology = override.Nil(c.Ontology, other.Ontology)
-	c.Group = override.Nil(c.Group, other.Group)
-	return c
-}
-
+// A Service is how users are managed in the Synnax cluster.
 type Service struct {
+	// Config is the configuration for the service.
 	Config
 	group group.Group
 }
 
 const groupName = "Users"
 
-func NewService(ctx context.Context, configs ...Config) (*Service, error) {
-	cfg, err := config.New(DefaultConfig, configs...)
+// OpenService opens a new Service with the given context ctx and configurations configs.
+func OpenService(ctx context.Context, configs ...Config) (*Service, error) {
```

Comment:
This function was correctly named `NewService` as it does not need to be closed after use. `New` is used for constructors that do not need to be closed, `Open` is used for constructors that need a call to `Close` in order to be shutdown.